toolbox/repo/validate_role_files.py

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env python
+
+import glob, os
+import yaml
+import pathlib
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+
+# go to top directory
+os.chdir(THIS_DIR.parent.parent)
+
+ROLES_VARS_GLOB = "roles/*/vars/*/*"
+FILE_PREFIX = "roles/"
+
+def validate_role_vars_files(filename, yaml_doc):
+    errors = 0
+
+    for key, value in yaml_doc.items():
+        try:
+
+            if not value.startswith(FILE_PREFIX):
+                print(f"{key}: {value} --> not starting with '{FILE_PREFIX}', ignoring.")
+                continue
+        except AttributeError: # value.startswith
+            print(f"{key}: --> no a string, ignoring.")
+            continue
+
+        if not pathlib.Path(value).exists:
+            errors += 1
+
+            print(f"ERROR: {key}: {value} --> not found")
+            continue
+
+        # file found, nothing to do
+
+    return errors
+
+def traverse_role_vars():
+    errors = 0
+    for filename in glob.glob(ROLES_VARS_GLOB):
```

Comment:
nit: If you're already importing pathlib, you can use its built-in globing functionality like ```python
for path in pathlib.Path(FILE_PREFIX).glob("*/vars/*/*"):
```